R/write.R

```diff
@@ -17,11 +17,9 @@
 #' @details
 #'   * Variable and dataset labels are stored in the "label" attribute.
 #'
-#'   * SAS length are stored in the "SASlength" attribute.
-#'
 #'   * SAS format are stored in the "SASformat" attribute.
```

Comment:
The name for the SAS format attribute is 'format.sas'. Maybe we should also update 'SASformat' to  'format.sas', what do you think?